jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/annotations/PostConstructAnnotation.java

```diff
@@ -1,32 +1,63 @@
 package io.openliberty.sample.jakarta.annotations;
 
+import java.io.IOException;
+
 import jakarta.annotation.PostConstruct;
 import jakarta.annotation.Resource;
 
 @Resource(type = Object.class, name = "aa")
 public class PostConstructAnnotation {
 
-    private Integer studentId;
+	private Integer studentId;
+
+	private boolean isHappy;
+
+	private boolean isSad;
+
+	private String emailAddress;
+
+	@PostConstruct()
+	public Integer getStudentId() {
+		return this.studentId;
+	}
+
+	@PostConstruct
+	public void getHappiness(String type) {
 
-    private boolean isHappy;
+	}
 
-    private boolean isSad;
+	@PostConstruct
+	public void throwTantrum() throws Exception {
+		System.out.println("I'm sad");
+	}
 
-    @PostConstruct()
-    public Integer getStudentId() {
-        return this.studentId;
-    }
+	@PostConstruct
+	public void throwRuntimeException() throws RuntimeException {
+		System.out.println("RuntimeException");
+	}
 
-    @PostConstruct
-    public void getHappiness(String type) {
+	@PostConstruct
+	public void throwNullPointerException() throws NullPointerException {
+		System.out.println("NullPointerException");
+	}
 
-    }
+	@PostConstruct
+	public void throwIOException() throws IOException {
+		System.out.println("IOException");
+	}
 
-    @PostConstruct
-    public void throwTantrum() throws Exception {
-        System.out.println("I'm sad");
-    }
+	@PostConstruct
+	public void throwCustomCheckedException() throws CustomCheckedException {
+		System.out.println("CustomCheckedException");
+	}
 
-    private String emailAddress;
+	@PostConstruct
```

Comment:
Here in PostConstructAnnotation, these two last methods (throwCustomUnCheckedException, throwError) were added, but appear not to be tested. Please test them.
Same comment applies to  PreDestroyAnnotation.java